jp/components/navbar.md

```diff
@@ -105,31 +105,11 @@ export class AppModule {}
 
 <div class="divider"></div>
 
-### API まとめ
+### API References
 
-このトピックでは NavBar コンポーネントの使用例を紹介しました。以下は、使用したコンポーネントの API です。
+In this article we show a few scenarios where the navbar component may come in handy. The APIs, we used to achieve them, are listed in the links below.
 
-#### 入力
-
-以下の入力が **igx-navbar** コンポーネントで利用できます。
-
-| 名前 | 型 | 説明 |
-| :--- | :--- | :--- |
-| `actionButtonIcon` | String | アクション ボタンに使用するマテリアル アイコンの名前を提供します。 |
-| `isActionButtonVisible` | Boolean | アクション ボタンが表示されるかどうか。 |
-| `title` | String | NavBar に表示されるタイトルを提供します。 |
-
-<div class="divider"></div>
-
-#### 出力
-
-以下の出力が **igx-navbar** コンポーネントで利用できます。
-
-| 名前 | 型 | 説明 |
-| :--- | :--- | :--- |
-| `onAction` | EventEmitter | アクションが実行されたときにイベントを発生します。 |
-
-<div class="divider"></div>
+[`IgxNavbarComponent`]({environment:angularApiUrl}/classes/igxnavbarcomponent.html)
```

Comment:
Same request changes as in the [English version](https://github.com/IgniteUI/igniteui-docfx/pull/731/files#diff-8d520a0587de247cd5ae85f0fe18ee46R104)